config/systemConfig/java.yaml

```diff
@@ -9,4 +9,4 @@ systemConfig:
     value: (?i)(?:url|client|openConnection|request|execute|newCall|load|host|access|fetch|get|getInputStream|getApod|getForObject|getForEntity|list|set|put|post|proceed|trace|patch|Path|send|sendAsync|remove|delete|write|read|assignment|provider|exchange|postForEntity)
 
   - key: apiIdentifier
-    value: (?i).*((hook|base|authorize|provider|endpoint|installation)(s){0,1}(_){0,1}url|(slack|web)(_){0,1}hook|(rest|api)(_){0,1}endpoint).*
\ No newline at end of file
+    value: ^(?i).*((hook|base|auth|prov|endp|install|request|service|gateway|route|resource)(.){0,5}url|(slack|web)(.){0,4}hook|(rest|api|request|service)(.){0,4}(endpoint|gateway|route)).*
```

Comment:
can we increase the {0,5} limit here : `ateway|route|resource)(.){0,5}url|(slack|we` seems longer variables like authorisationUrl wont match